README.md

```diff
@@ -0,0 +1,27 @@
+# Segtree
```

Comment:
Please use the full name i.e. k-ary segtree or more concisely k-child-per-node segmented b-tree.